developer-docs/platform_introduction.md

```diff
@@ -217,70 +217,76 @@ $ minikube ip
 
 The same configuration you just created can be used to deploy the application to any of the clusters such as CI, Dev and Prod.
 
-For induction sessions we will be deploying to the **dev-induction** namespace. Typically you would be using one of
-your project namespace.
+In this particular scenario you can deploy your service to the **dev-induction** namespace. Typically you would be using one of your project namespace.
 
-You will need to update your kubernetes context - this determines which cluster you are talking to, which token to use,
-and which namespace you are using. You can view your current contexts with:
+You will need to update your kubernetes context - this determines which cluster you are talking to, which token to use, and which namespace you are using. You can view your current contexts with:
 
-```
-kubectl config view
+```bash
+$ kubectl config view
 ```
 
-If you have a context set with the dev cluster (https://kube-dev.dsp.notprod.homeoffice.gov.uk) and the dev-induction namespace
- change to that context with:
+If you have a context set with the dev cluster (https://kube-dev.dsp.notprod.homeoffice.gov.uk) and the _dev-induction_ namespace change to that context with:
 
-```
-kubectl config use-context <context-name>
+```bash
+$ kubectl config use-context <context-name>
 ```
 
-If you don't have this context you will need to set it with something like:
-```
-kubectl config set-context dev-induction --cluster=dsp-dev --user=dsp-dev --namespace=dev-induction
+If you don't have this context you can set it with:
+
+```bash
+$ kubectl config set-context dev-induction \
+          --cluster=dsp-dev \
+          --user=dsp-dev \
+          --namespace=dev-induction
 ```
 
-To ensure people on the induction don't have name clashes with their deployed applications we are going to use a
-version of our deployment files where the application name has been templated out. [kd](https://github.com/UKHomeOffice/kd)
-is a tool that wraps kubectl and enables us to do this templating.
+To ensure people on the induction don't have name clashes with their deployed applications we are going to use a version of our deployment files where the application name has been templated out. [kd](https://github.com/UKHomeOffice/kd) is a tool that wraps kubectl and enables us to do this templating.
 
-Please specify a unique APP_NAME with your initials and some random characters when you deploy. You will also noted the image
-version is templated out - a very common pattern. Please use v1 for the version.
+Please specify a unique `APP_NAME` with your initials and some random characters when you deploy. You will also noted the image version is templated out - a very common pattern. Please use _v1_ for the version.
 
 ```bash
-APP_NAME=tgxu172 APP_VERSION=v1 kd --file kube-templated/deployment.yaml --file kube-templated/service.yaml --file kube-templated/ingress.yaml
+$ APP_NAME=tgxu172 \
+  APP_VERSION=v1 \
+  kd --file kube-templated/deployment.yaml \
```

Comment:
presuming you're using https://github.com/UKHomeOffice/dsp-hello-world/blob/master/kube-templated/deployment.yaml it'll depend on the secret first